README-ANSIBLE_AI_MODEL_MESH_CONFIG.md

```diff
@@ -0,0 +1,40 @@
+# Example `ANSIBLE_AI_MODEL_MESH_CONFIG` configuration
+
+Pay close attention to the formatting of the blocks.
+
+Each ends with `}},` otherwise conversion of the multi-line setting to a `str` can fail.
```

Comment:
YAML would maybe be more elegant here?

Reply:
@goneri I've added support for loading _either_ JSON or YAML and updated the README's.

Reply:
"Pay close attention to the formatting of the blocks".

This was caused by (me) trying to define JSON as a multi-line String in a `.env` file.

Moving it to a Django/Python settings file and re-configuring my local "launch" configuration fixed things.